conda-forge.yml

```diff
@@ -1,6 +1,6 @@
 bot:
   automerge: true
-  inspect: update-grayskull
+  inspection: update-grayskull
```

Comment:
I did these changes here based on the json template in conda-smithy 3.32 as otherwise the recipe was marked as broken. also linux_aarc64 and linux_ppc64le keys below are gone.